certificate-policy.md

```diff
@@ -654,6 +660,10 @@ Certificate applications SHALL be processed and a certificate issued within 30 d
 ### 4.3.1 CA actions during certificate issuance
 Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
 
+All end entity certificates asserting the id-kp-serverAuth EKU SHALL assert a Certificate Transparency (CT) Signed Certificate Timestamp (SCT) via a certificate extension. At a minimum, the CA SHALL request the following number of SCTs:
+- One embedded SCT from a Google CT log
+- One embedded SCT from non-Google CT log
```

Comment:
:+1: But it's worth noting that by the time anything is in operation and undergoing an audit, Mozilla may have finalized a new CT policy, and/or Google may have changed theirs in tandem with their intended CT-for-everything 2017 policy.